src/styles/main.scss

```diff
@@ -1,3 +1,33 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./blocks/stars.scss";
+@import "./blocks/cards.scss";
+@import "./blocks/header.scss";
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.catalog__cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  padding-top: 50px;
+  padding-bottom: 71px;
+}
+
+.card:nth-child(-n+3) {
+  margin-right: 48px;
+}
+
+.row:nth-child(-n+1) {
+  margin-bottom: 46.5px;
 }
```

Comment:
if this styles for blocks - it should have separate file, if it's element inside some block, it should have corresponding name